homeassistant/components/vizio/strings.json

```diff
@@ -4,12 +4,14 @@
         "step": {
             "user": {
                 "title": "Setup Vizio SmartCast Device",
-                "description": "All fields are required except Access Token. If you choose not to provide an Access Token, and your Device Type is 'tv', you will go through a pairing process with your device so an Access Token can be retrieved.\n\nTo go through the pairing process, before clicking Submit, ensure your TV is powered on and connected to the network. You also need to be able to see the screen.",
+                "description": "If you are configuring a TV:\n - If it supports apps, you can choose to whitelist or blacklist apps by choosing Include or Exclude correspondingly and selecting apps from the app list.\n - If you choose not to provide an Access Token, and your Device Type is 'tv', you will go through a pairing process with your device so an Access Token can be retrieved. To go through the pairing process, before clicking Submit, ensure your TV is powered on and connected to the network. You also need to be able to see the screen.",
```

Comment:
This description is way too long. Let's break it up in 2 pages.

Reply:
Will do